spec/openhab/dsl/rules/builder_spec.rb

```diff
@@ -1019,6 +1035,24 @@ def self.generate(name, cron_expression, *every_args, attach: nil, **kwargs)
                at: LocalTime.parse("12:00"))
       generate("can use MonthDay as a string", "0 0 12 17 11 ? *", "11-17", at: LocalTime.parse("12:00"))
       generate("can use LocalTime a string", "0 0 12 17 11 ? *", MonthDay.parse("11-17"), at: "12:00")
+
+      it "supports dynamic `at`" do
+        items.build { date_time_item MyDateTimeItem }
+
+        triggered = false
+        every(:day, at: MyDateTimeItem) { triggered = true }
+
+        MyDateTimeItem.update(Time.now + 2 - 2.days)
+        wait(4.seconds) do
+          expect(triggered).to be true
+        end
+      end
+
+      it "complains about dynamic at that's not daily" do
+        items.build { date_time_item MyDateTimeItem }
+
+        expect { every :month, at: MyDateTimeItem }.to raise_error(ArgumentError)
```

Comment:
https://ccutrer.github.io/openhab-jrubyscripting/main/OpenHAB/DSL/Rules/BuilderDSL.html#every-instance_method

`every :monday, at: '5:15'` is supported, why not allow `:month`, `:week`, etc)?

Reply:
because the underlying trigger doesn't support that, and I don't have interest in doing extra work to support it here.